etc/omero.properties

```diff
@@ -868,6 +862,9 @@ omero.client.viewer.roi_limit=2000
 # sorting them by default client ordering strategy
 omero.client.ui.tree.type_order=tagset,tag,project,dataset,screen,plate,acquisition,image
 
+# The default thumbnail size
+omero.client.browser.thumb_default_size=96
```

Comment:
You have to define a bean for new property in https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/common/resources/ome/config.xml